src/libcprover-cpp/verification_result.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2023 Fotis Koutoulakis for Diffblue Ltd.
+
+/// \file
+/// Interface for the various verification engines providing results in a structured form.
+
+#ifndef CPROVER_GOTO_CHECKER_VERIFICATION_RESULT_H
+#define CPROVER_GOTO_CHECKER_VERIFICATION_RESULT_H
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+class dstringt;
+struct property_infot;
+using propertiest = std::map<dstringt, property_infot>;
+enum class resultt;
```

Comment:
:pick: Seems like quite a generic symbol to introduce into the public global namespace.